pkg/controller/certificates/signer/config/v1alpha1/conversion.go

```diff
@@ -26,15 +26,15 @@ import (
 // with CSRSigningControllerConfiguration types need to be manually exposed like this in order for
 // other packages that reference this package to be able to call these conversion functions
 // in an autogenerated manner.
-// TODO: Fix the bug in conversion-gen so it automatically discovers these Convert_* functions
+// TODO: Fix the bug in conversion-gen so it automatically discovers these Convert* functions
 // in autogenerated code as well.
 
-// Convert_v1alpha1_CSRSigningControllerConfiguration_To_config_CSRSigningControllerConfiguration is an autogenerated conversion function.
-func Convert_v1alpha1_CSRSigningControllerConfiguration_To_config_CSRSigningControllerConfiguration(in *v1alpha1.CSRSigningControllerConfiguration, out *csrsigningconfig.CSRSigningControllerConfiguration, s conversion.Scope) error {
+// ConvertV1alpha1CSRSigningControllerConfigurationToConfigCSRSigningControllerConfiguration is an autogenerated conversion function.
```

Comment:
We have a convention for naming conversion function which explicitly states what you're converting from which version to which version. I'd prefer we keep that convention in-place rather than following golint complains.

Reply:
Thanks for pointing that out, I was unable to find any documentation for it. Is this relevant documentation https://godoc.org/k8s.io/code-generator/cmd/conversion-gen ?